addons/binding/org.openhab.binding.bosesoundtouch10/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BoseSoundTouch10 Binding
+Bundle-SymbolicName: org.openhab.binding.bosesoundtouch10;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ javax.jmdns;version="3.5.1",
```

Comment:
Do you need those specific versions of the jmdns and jetty libraries? If not, please remove the version restrictions.